test/connect.test.js

```diff
@@ -73,6 +73,7 @@ describe('connection model connector', () => {
         assert(model.isValid());
         mockConnect(model, setupListeners, err => {
           // must throw error here, because the connection details are invalid
+          console.log(err);
```

Comment:
Do we want to keep the console log here?

Reply:
Whoops! :)